Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -162,6 +161,16 @@ for multiplication in a normed algebra over the base field. -/
 theorem differentiable_circleMap (c : ℂ) (R : ℝ) : Differentiable ℝ (circleMap c R) := fun θ =>
   (hasDerivAt_circleMap c R θ).differentiableAt
 
+open Complex in
+/-- The circleMap is real analytic. -/
+theorem analyticOnNhd_circleMap {c : ℂ} {R : ℝ} :
```

Comment:
All the other statements with the same flavor have `c` and `r` explicit, so could you do the same?

Reply:
Implemented.